Apps/Examples/Examples/SiwftUI Examples/FeaturesQueryExample.swift

```diff
@@ -0,0 +1,112 @@
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct FeaturesQueryExample: View {
+    @State private var camera = CameraState(center: .newYork, zoom: 10)
+
+    @State private var queryResult: QueryResult? = nil
+
+    var body: some View {
+            Map(camera: $camera)
+                .styleURI(.streets)
+                .onMapTapGesture(action: { _, res in
```

Comment:
Consider renaming `action` to `perform` to be consistent with SwiftUI?
https://developer.apple.com/documentation/swiftui/view/ontapgesture(count:perform:)

Reply:
And probably `res` to `result`, I thought it means `resources` at the first glance😅